roles/engineblock/defaults/main.yml

```diff
@@ -80,6 +80,9 @@ engine_fpm_memory: 128M
 # When using vagrant for provisioning, it's not possible to put something in the inventory dir
 engine_crt_not_in_inventory: false
 
+# The memory limit used for the metadata push
+metadata_push_memory_limit: 256M
+
```

Comment:
Could you prefix this variable with engine_ ? It makes it easier to recognise if you override it somewhere

Reply:
@quartje done